cmd/clusterctl/client/repository/components.go

```diff
@@ -292,9 +301,73 @@ func addNamespaceIfMissing(objs []unstructured.Unstructured, targetNamespace str
 	return objs
 }
 
+func fixWebHookTargetNamespace(o unstructured.Unstructured, targetNamespace string) (unstructured.Unstructured, error) {
```

Comment:
Can you move this func below fixTargetNamespace? (just easier to read top-down imho)